Sources/XcodeProj/Scheme/XCScheme+TestableReference.swift

```diff
@@ -33,10 +33,34 @@ public extension XCScheme {
             self.selectedTests = selectedTests
             self.skippedTests = skippedTests
         }
+      
+        public init(skipped: Bool,
```

Comment:
Let's mark this `init` deprecated with a suggestion how to migrate.

Reply:
Let me know how that reads, as it follows the usages found elsewhere in the codebase.